TombEngine/Sound/sound.h

```diff
@@ -38,6 +38,18 @@ enum class SoundTrackType
 	Count
 };
 
+enum SampleRate
+{
+	TRLE_QUALITY_SAMPLES = 22050,
+	HIGH_QUALITY_SAMPLES = 44100,
+};
+
+enum Channels
```

Comment:
Sound samples which are playing in a 3D space, which is essentially all the sound engine does, cannot be stereo. Therefore, this enum makes no sense as well.